go-controller/pkg/ovn/policy.go

```diff
@@ -605,12 +597,18 @@ func (oc *Controller) createMulticastAllowPolicy(ns string, nsInfo *namespaceInf
 		}
 	}
 
+	// for the update case need to delete the old style of multicast port group
+	// otherwise a stale port_group would stick around with no way to edit it
+	// make sure to delete the old port_group after the new one is created and
+	// ACLs are applied for minimal downtime.
+	deletePortGroup(hashedPortGroup(fmt.Sprintf("mcastPortGroup-%s", ns)))
```

Comment:
The old ACL entries would still have the old hashed port group name though... does something ensure that new ACLs get written out with the new name?

Reply:
do the ACLs stick around when a port group is deleted? Unless I am reading this wrong the delete of the old port group happens after a whole set has been recreated just above this delete

Reply:
also changed this to clean up the stale ACLs